app/src/main/res/values/strings.xml

```diff
@@ -59,4 +59,6 @@
     <string name="modify_profile_icon">Modify profile icon</string>
     <string name="gmail_icon">Gmail icon</string>
     <string name="photo_cluster_item">Photo of cluster item</string>
+    <string name="occupancyClear">Not busy</string>
```

Comment:
Will you use this string in the future? It appears to be unused.

Reply:
Removed !